src/mrnet/core/mol_entry.py

```diff
@@ -59,8 +59,9 @@ def __init__(
         attribute=None,
         mol_doc: Optional[Dict] = None,
         mol_graph: Optional[MoleculeGraph] = None,
+        dummy: Optional[bool] = False,
     ):
-
+        self.dummy = dummy
```

Comment:
I'm going to go ahead and revert the mol_entry class to its original state (for this PR at least), since I was doing that in preparation for the Thermo Entry class. Since that is going to take a backseat and is nowhere near finished, it doesn't seem appropriate for these half baked changes to get merged in. Thanks for catching this!